helm-v3/001-charts.md

```diff
@@ -19,8 +19,8 @@ be found below.
 
 ### Sandboxing Model
 
-When a chart is loaded, all of the scripts in `ext/lua` will also be loaded. All
-chart and subchart scripts will be loaded into the same runtime, and then the
+When a chart is loaded, the script `chart.lua` in `ext/lua` will also be loaded. All
+subchart `ext/lua/chat.lua` scripts will be loaded into the same runtime, and then the
```

Comment:
`ext/lua/chat.lua`? .... notice the chat instead of chart... are you putting a chat bot in your chart?

Reply:
LOL... you caught me! Next I'm going to switch it to `bitcoin.lua`.

Reply:
Fixed